facturacion_mexico/cfdi_recibidos/services/wizard_cfdi_recibidos.py

```
@@ -0,0 +1,272 @@
+	config.ultima_generacion = now_datetime()
+	config.save(ignore_permissions=True)
+
+	return {
+		"status": "ok",
+		"template_name": template_name,
+		"warnings": warnings,
+		"message": _("Template '{0}' generado correctamente.").format(template_name),
+	}
+
+
+def _collect_warnings(config, active_rules: list) -> list:
```

Ruff (0.15.14)

[warning] Unused function argument: `config`

(ARG001)